app/marketing/management/commands/new_bounties_email.py

```diff
@@ -29,17 +29,16 @@
 def get_bounties_for_keywords(keywords, hours_back):
     new_bounties_pks = []
     all_bounties_pks = []
-    for keyword in keywords:
-        relevant_bounties = Bounty.objects.filter(
-            network='mainnet',
-            current_bounty=True,
-            metadata__icontains=keyword,
-            idx_status__in=['open'],
-            )
-        for bounty in relevant_bounties.filter(web3_created__gt=(timezone.now() - timezone.timedelta(hours=hours_back))):
-                new_bounties_pks.append(bounty.pk)
-        for bounty in relevant_bounties:
-                all_bounties_pks.append(bounty.pk)
+    relevant_bounties = Bounty.objects.filter(
+        network='mainnet',
+        current_bounty=True,
+        idx_keywords__ioverlap=keywords, #TODO: do we search the title/desc too?
```

Comment:
@owocki I think we need to shift from `ArrayField` keywords to taggit, honestly.  That would allow all us to simply relate keywords with `Profile`, `Organization`, and `Bounty` objects.  Title and description matching might get weird.  If we establish user expectation that `keywords` will indeed attract the appropriate hunters/funders, it shouldn't be an issue and would make landings, explorer filtering/sorting, and skill/bounty matching much simpler.

Reply:
i agree with that direction for the long term; but whether we merge this in the short term depends upon how far out that is in the roadmap.  right now people are getting emails every day with bounties that seemingly don't match with their preferences -- so there is urgency in doing something soon